platform/darwin/src/MGLStyle.mm

```diff
@@ -122,6 +122,8 @@ + (NSURL *)name##StyleURL##WithVersion:(NSInteger)version { \
 static_assert(6 == mbgl::util::default_styles::numOrderedStyles,
               "mbgl::util::default_styles::orderedStyles and MGLStyle have different numbers of styles.");
 
+@synthesize accessiblePlaceSourceIdentifiers = _accessiblePlaceSourceIdentifiers;
```

Comment:
It shouldn’t be necessary to synthesize this property explicitly.

Reply:
Thanks, I think that's leftover from when I was taking a different approach! Will clean it up.